x/ccv/provider/keeper/throttle.go

```diff
@@ -165,10 +165,9 @@ func (k Keeper) ReplenishSlashMeter(ctx sdktypes.Context) {
 func (k Keeper) GetSlashMeterAllowance(ctx sdktypes.Context) sdktypes.Int {
 
 	strFrac := k.GetSlashMeterReplenishFraction(ctx)
-	decFrac, err := sdktypes.NewDecFromStr(strFrac)
-	if err != nil {
-		panic(fmt.Sprintf("failed to parse slash meter replenish fraction: %s", err))
-	}
+	// The below func should not panic, since the (string representation) of the slash meter replenish fraction
+	// is validated in ValidateGenesis and anytime the param is mutated.
+	decFrac := sdktypes.MustNewDecFromStr(strFrac)
```

Comment:
```suggestion
	// MustNewDecFromStr should not panic, since the (string representation) of the slash meter replenish fraction
	// is validated in ValidateGenesis and anytime the param is mutated.
	decFrac := sdktypes.MustNewDecFromStr(strFrac)
```